pom.xml

```diff
@@ -75,6 +75,11 @@
             <name>Francesco Caliumi</name>
             <organization>Diennea</organization>
         </developer>
+        <developer>
+            <id>amitvc</id>
+            <name>Amit Chavan</name>
+            <organization>Diennea</organization>
```

Comment:
I think this is not a good value, as you are not an "employee" of Diennea.

This field is optional, you can drop the <organization> tag

Reply:
Ok will do.